marketplace-listing/src/graphql/queries/marketplaceListings.graphql

```diff
@@ -0,0 +1,55 @@
+query MarketplaceListings(
```

Comment:
Imo, this is going to become too slow super fast. We should restructure this a bit to follow the doc and how most apps actually do it. 

Listings should _just_ pull in the models for sale. Then, the next page should let you pick the item to get.

Reply:
Listings now shows a list of all marketplace listings and also paginates with "Show More" button
<img width="1428" alt="image" src="https://user-images.githubusercontent.com/34705337/227269295-a554f13a-3981-40dc-8d8c-d9bf902b0205.png">